src/serve/dashboard/common/functions.ts

```diff
@@ -0,0 +1,9 @@
+import { blake32Hash } from '../deps.ts'
+import { Buffer } from 'node:buffer'
+
+if (typeof window === 'object' && typeof Buffer === 'undefined') {
+  ;(window as unknown as { Buffer: typeof Buffer }).Buffer = Buffer
```

Comment:
Two comments here:

1. I believe earlier I said that `npm:buffer` shouldn't be used. I forgot about the dashboard frontend, which runs in the browser. `npm:buffer` should be used here, since `node:buffer` is a Node.js / Deno internal.
2. In GI, outside of the dashboard, this has been replaced by importing `Buffer` where needed. I've searched for uses of `Buffer` in `src/serve/dashboard`, and it looks like `Buffer` isn't actually used, so this can in fact likely be removed.